justfile

```diff
@@ -101,10 +104,11 @@ generate cli=default_cli platform=default_platform:
     nix run -L .#{{ cli }} -- generate \
         --workspace-dir ./{{ workspace_dir }} \
         --image-replacements ./{{ workspace_dir }}/just.containerlookup \
-        --reference-values {{ platform }}\
+        --reference-values {{ platform }} \
+        --genpolicy-cache-path ./{{ workspace_dir }}/../layers-cache.json \
         ./{{ workspace_dir }}/deployment/*.yml
 
-    # On baremetal SNP, we don't have default values for MinimumTCB, so we need to set some here.
+    # On baremetal SNP, we do not have default values for MinimumTCB, so we need to set some here.
```

Comment:
Why was this changed?

Reply:
The `'` broke my syntax highlighting, so I changed it temporarily. Since it's unrelated I'll revert this.